config/initializers/productization.rb

```diff
@@ -1 +1,3 @@
-Vmdb::Productization.new.prepare
+Rails.application.config.to_prepare do
```

Comment:
Well, before, we were adding our path before `railsties` had a chance. So they were adding themselves before us and productization could not be performed.

We just need to make sure we are added after `railsties` gem has modified the path.